app/templates/orders/order_transport.hbs

```diff
@@ -81,6 +81,24 @@
         </div>
       </div>
 
+      <div class="row">
+        <div class="small-5 large-4 columns">
+          Client Name:
+        </div>
+        <div class="small-7 large-8 columns">
+          {{model.orderTransport.contact.name}}
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="small-5 large-4 columns">
+          Client Mobile:
+        </div>
+        <div class="small-7 large-8 columns">
+          {{model.orderTransport.contact.mobilePhoneNumber}}
```

Comment:
Can we use **phoneNumber** here? what's the difference between **phoneNumber** and **mobilePhoneNumber** field?